api/proto/external.proto

```diff
@@ -31,9 +24,12 @@ message CompressedRistretto {
 }
 
 /// A 32-byte scalar associated to the ristretto group.
-/// This is the same as RistrettoPrivate, but they are used in different places,
-/// the transaction crate generally uses CurveScalar and higher-level crates use
-/// RistrettoPrivate.
+/// This is the same as RistrettoPrivate, but they are used in different places.
+/// Many crates only need keys to perform key exchange operations, and strictly
+/// use the RistrettoPublic and RistrettoPrivate types.
+/// The transaction crate often uses CurveScalar and sometimes must perform
+/// raw elliptic curve operations etc.
+/// This type can possibly be factored out.
```

Comment:
I think we should leave this out since this file is mainly meant to be used as an interface file. Suggestions about improvements should probably take the form of a comment on the PR or a dedicated Jira ticket.

It's also not clear to me whether this is a confirmed statement or just a thought that needs more investigation.

Reply:
created MC-1605 to track this and removed commentary from the proto comment